client/ayon_core/pipeline/farm/pyblish_functions.py

```diff
@@ -295,11 +295,17 @@ def _add_review_families(families):
     return families
 
 
-def prepare_representations(skeleton_data, exp_files, anatomy, aov_filter,
-                            skip_integration_repre_list,
-                            do_not_add_review,
-                            context,
-                            color_managed_plugin):
+def prepare_representations(
+    skeleton_data,
+    exp_files,
+    anatomy,
+    aov_filter,
+    skip_integration_repre_list,
+    do_not_add_review,
+    context,
+    color_managed_plugin,
+    frames_to_render
```

Comment:
Should we make this new key optional for better backwards compatibility (if anyone was using it outside of us?)?

Reply:
I 100% agree.